lib/Schema.js

```diff
@@ -87,13 +87,24 @@ function Schema(obj, options) {
 Schema.prototype.toDynamo = function(model) {
 
   var dynamoObj = {};
-  //debug('toDynamo with schema attributes', this.attributes);
-  for(var name in this.attributes) {
+
+  for(var name in model) {
+    if(!model.hasOwnProperty(name)){
+      continue;
+    }
     var attr = this.attributes[name];
-    attr.setDefault(model);
-    var dynamoAttr = attr.toDynamo(model[name], undefined, model);
-    if(dynamoAttr) {
-      dynamoObj[attr.name] = dynamoAttr;
+
+    if(!attr & this.options.saveUnknown) {
+      attr = Attribute.create(this, name, typeof model[name]);
```

Comment:
I think the `typeof` here works better than always assuming `Object`.  `JSON.stringify` doesn't always give you what you'd want. 
```
> JSON.stringify("astring")
'"astring"'
```